.vscode/settings.json

```diff
@@ -2,7 +2,7 @@
   "files.exclude": {
     "node_modules/": true
   },
-  "editor.formatOnSave": true,
+  "editor.formatOnSave": false,
```

Comment:
Sorry forgot I had disabled that, it was causing more problems than solutions because my formatter is set up very differently. Didn't mean to commit it, just left my mind.

Reply:
All good! I get it!